sdk/spring/azure-spring-boot-starter-keyvault-secrets/README.md

```diff
@@ -28,6 +28,68 @@ For highly sensitive data, clients should consider additional layers of protecti
+spring.cloud.azure.keyvault.secret-service-version|Valid secret-service-version value can be found [here][version_link].|The latest value|
+spring.cloud.azure.keyvault.uri|Azure Key Vault Uri.||
+
+Note: for multiple Key Vault usage, specify your Key Vault name after the prefix of `spring.cloud.azure.keyvault.`. For usage examples, please refer to section [Multiple Key Vault support](#multiple-key-vault-support-1).
+#### Deprecated Properties
+|Obsolete Type|Description|Default Value|Obsolete Type|
+|:---|:---|:---|:---
+azure.keyvault.authority-host|Authority Host URI|https://login.microsoftonline.com/ |**spring.cloud.azure.keyvault.environment.authority-host** |
+azure.keyvault.case-sensitive-keys|Defines the constant for the property that enables/disables case sensitive keys.|false|**spring.cloud.azure.keyvault.case-sensitive-keys** |
+azure.keyvault.certificate-password|Password of the certificate file to use when performing service principal authentication with Azure.| |**spring.cloud.azure.keyvault.credential.client-certificate-password** |
+azure.keyvault.certificate-path|Path of a PEM certificate file to use when performing service principal authentication with Azure.| |**spring.cloud.azure.keyvault.credential.client-certificate-path** |
+azure.keyvault.client-id|Client id to use when performing service principal authentication with Azure.| |**spring.cloud.azure.keyvault.credential.client-id** |
+azure.keyvault.client-key|Client secret to use when performing service principal authentication with Azure.| |**spring.cloud.azure.keyvault.credential.client-secret** |
+azure.keyvault.enabled|To turn on/off Azure Key Vault Secret property source.|true|**spring.cloud.azure.keyvault.enabled** |
+azure.keyvault.order|Define the order of the key vaults you are delivering (comma delimited, e.g 'my-vault, my-vault-2').| |**spring.cloud.azure.keyvault.order** |
+azure.keyvault.refresh-interval|Interval for PropertySource to refresh secret keys|1800000(ms) |**spring.cloud.azure.keyvault.refresh-interval** |
+azure.keyvault.secret-keys|If application using specific secret keys. If this property is set, application will only load the keys in the property and won't load all the keys from keyvault, that means if you want to update your secrets, you need to restart the application rather than only add secrets in the keyvault.| |**spring.cloud.azure.keyvault.secret-keys** |
+azure.keyvault.secret-service-version|Valid secret-service-version value can be found [here][version_link].|The latest value |**spring.cloud.azure.keyvault.secret-service-version** |
+azure.keyvault.tenant-id|Tenant id for the Azure resources.| |**spring.cloud.azure.keyvault.credential.tenant-id** |
+azure.keyvault.uri|Azure Key Vault Uri.| |**spring.cloud.azure.keyvault.uri** |
```

Comment:
URI and Uri, maybe we should make them consistent.

Reply:
Updated for URI now.